Document/0x05e-Testing-Cryptography.md

```diff
@@ -264,7 +288,12 @@ $ grep -r "Ljavax\crypto\spec\SecretKeySpec;"
 ```
 This will highlight all the classes that use the ```SecretKeySpec``` class, we now examine all the highlighted files and trace which bytes are used to pass the key material. The figure below shows the result of performing this assessment on a production ready application. For sake of readability we have reverse engineered the DEX bytecode to Java code. We can clearly locate the use of a static encryption key that is hardcoded and initialized in the static byte array ```Encrypt.keyBytes```.
 
-![Use of a static encryption key in a production ready application.](Images/Chapters/0x5e/static_encryption_key.png)
+![Use of a static encryption key in a production ready application.](Images/Chapters/0x5e/static_encryption_key.png).
+
+When you have access to the sourcecode, check at least for the following:
+- Check which mechanism is used to store a key: prevail the `AndroidKeyStore` over other solutions.
```

Comment:
prevail the `AndroidKeyStore` over other solutions. (prevail is an intransitive verb so it cannot have a direct object; i.e. you cannot "prevail something" but you can "prefer something" (https://www.merriam-webster.com/dictionary/intransitive) )
Alt.1 prefer the `AndroidKeyStore` over other solutions.
Alt.2 the `AndroidKeyStore` prevails over other solutions.